src/components/alarmList/alarmList.js

```diff
@@ -55,51 +74,40 @@ class AlarmList extends Component {
   componentWillReceiveProps(nextProps) {
     // If the device list changes, reload the list data
     const newDeviceIdList = this.generateDeviceIdList(nextProps.devices);
-    if (this.state.deviceIdList !== newDeviceIdList ||
-      this.state.timeRange !== nextProps.timeRange) {
-      this.setState(
-        {
-          deviceIdList: newDeviceIdList,
-          rowData: undefined,
-          timeRange: nextProps.timeRange
-        }, 
-        () => this.refresh(`filterGroupRefresh`)
-      );
+    if (this.state.deviceIdList !== newDeviceIdList) {
+      this.setState({ deviceIdList: newDeviceIdList, rowData: undefined }, () => {
```

Comment:
If you are going to add new lines for the parameters, give each parameter a new line.